nls/root/strings.js

```diff
@@ -30,5 +31,8 @@ define({
     PREF_AUTOHIDE_NAME: "Outline - Autohide",
     PREF_AUTOHIDE_DESC: "True to enable autohide of the Outline List",
     PREF_AUTOHIDE_DELAY_NAME: "Outline - Autohide Delay",
-    PREF_AUTOHIDE_DELAY_DESC: "Milliseconds of delay for the Outline List to be exposed when the mouse is over the placeholder"
+    PREF_AUTOHIDE_DELAY_DESC: "Milliseconds of delay for the Outline List to be exposed when the mouse is over the placeholder",
+
+    AUTOHIDE_DELAY_MODAL_TITLE: "Outline List - Autohide Expose Delay",
+    AUTOHIDE_DELAY_MODAL_MS_LABEL: " milliseconds"
```

Comment:
Please format the HTML so we don't have to rely on the leading space in the translation strings.

Reply:
Sorry for the delay. I've been busy the last few days.

Fixed.